orca-spring-batch/src/main/java/com/netflix/spinnaker/zombie/ZombiePipelineCleanupConfig.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2017 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.zombie;
+
+import java.time.Duration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import redis.clients.jedis.JedisCommands;
+
+@Configuration
+@ComponentScan(basePackageClasses = ZombiePipelineCleanupConfig.class)
+public class ZombiePipelineCleanupConfig {
+
+  @Bean
+  Lock zombiePollerLock(JedisCommands redis) {
```

Comment:
How are you running it? If I execute the `Main` class from orca-web or run `gradle bootRun` it's working fine.